package.json

```diff
@@ -25,21 +25,22 @@
     "css-loader": "^1.0.0",
     "express": "^4.16.3",
     "express-graphql": "^0.6.12",
+    "express-pino-logger": "^3.0.2",
```

Comment:
Does `express-pino-logger` already log any http requests? If so we can probably remove logs like `list data sources request`

Reply:
Yes it does Peter, I will remove all cases in the resolvers where we do this